### PR TITLE
fix(translations): complete Chinese theme translations

### PR DESCRIPTION
**Pull request by SJFCS**
Completed untranslated portion

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

**Comment by slorber** [comment]
Thanks

@Josh-Cena let me know if you can review

**Comment by slorber** [comment]
(we need the CLA signature to merge this PR in case you didn't already sign it - the bot can take some time)

**Comment by SJFCS** [comment]
> (we need the CLA signature to merge this PR in case you didn't already sign it - the bot can take some time)

Three little things ago I have signed the CLA, but the robot did not pass

File: packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json [hunk 1]
```diff
@@ -67,6 +67,6 @@
   "theme.tags.tagsListLabel": "标签：",
   "theme.tags.tagsPageLink": "查看所有标签",
   "theme.tags.tagsPageTitle": "标签",
-  "theme.unlistedContent.message": "This page is unlisted. Search engines will not index it, and only users having a direct link can access it.",
-  "theme.unlistedContent.title": "Unlisted page"
+  "theme.unlistedContent.message": "此页面未列出。搜索引擎不会对其进行索引，只有拥有直接链接的用户才能访问。",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json:70]:
```suggestion
  "theme.unlistedContent.message": "此页面未列出。搜索引擎不会对其索引，只有拥有直接链接的用户才能访问。",
```

**Review by Josh-Cena — APPROVED** [review]
LGTM; one little suggestion. Don't worry about the bot; it can take very long...

File: packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json (hunk 1 above)
Comment (Kennysong55) [reply packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json:70]: Good

**Pull request merged**